bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Epoch.scala

```diff
@@ -0,0 +1,11 @@
+package edu.gemini.spModel.core
+
+/** Epoch in Julian years. */
+final case class Epoch(value: Double)
```

Comment:
Yes, I do recommend `final case class`, if only for the potential benefit of inlining and monomorphic method dispatch. I don't think it's a huge deal because as you say, they are "morally" final ... but there's no harm and it may help.

@cquiroz you can extend a case class with a non-case class. I don't know why this is allowed. I'll ask around.